src/lib/Behat/Page/UserSettingsPage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\AdminUi\Behat\Page;
+
+use Behat\Mink\Session;
+use Ibexa\AdminUi\Behat\Component\ContentActionsMenu;
+use Ibexa\Behat\Browser\Element\Criterion\ElementTextCriterion;
+use Ibexa\Behat\Browser\Locator\VisibleCSSLocator;
+use Ibexa\Behat\Browser\Page\Page;
+use Ibexa\Behat\Browser\Routing\Router;
+
+class UserSettingsPage extends Page
+{
+    private ContentActionsMenu $contentActionsMenu;
+
+    public function __construct(Session $session, Router $router, ContentActionsMenu $contentActionsMenu)
+    {
+        parent::__construct($session, $router);
+        $this->contentActionsMenu = $contentActionsMenu;
+    }
+
+    public function verifyIsLoaded(): void
+    {
+        $this->contentActionsMenu->verifyIsLoaded();
+        $this->getHTMLPage()->find($this->getLocator('title'))->assert()->textEquals('User Settings');
+    }
+
+    public function switchTab(string $tabName): void
+    {
+        $this->getHTMLPage()->find($this->getLocator($tabName))->click();
```

Comment:
We can use https://github.com/ibexa/admin-ui/blob/main/src/lib/Behat/Component/TableNavigationTab.php here, example:
https://github.com/ibexa/admin-ui/blob/6715a651fd2e5ae04b639444c87252ca7dec0a58/src/lib/Behat/Page/SystemInfoPage.php#L41-L44